### ISSUE TITLE
Graphics: Objects and embedded text of images should be described for assistive technology

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#82]
**Requirement**
Images should have an alternative text description that describes both the objects and the embedded text that the image contains, using the "alt" attribute.

**10 Failed Elements**
- [ ] 1. 
```html
<img class="navbar-logo_img" aria-label="button to go home" src="/assets/images/beaty-logo-horizontal-white.png" height="28">
```

- [ ] 2.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/icons/canadian-maple-leaf.svg">
```

- [ ] 3.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/icons/georeferenced-icon.svg">
```

- [ ] 4.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/icons/citations-icon.svg">
```

- [ ] 5.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/icons/image-icon.svg">
```

- [ ] 6.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/images/sec-card-image.jpg" srcset="https://api.gbif.org/v1/image/nOJfkaHG7pSGN8zU9d2sAFSrrOs=/fit-in/2000x0/https%3A%2F%2Fbeaty-biodiversity-museum.hp.gbif-staging.org%2F%2Fasset...
```

- [ ] 7.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/images/ctc-card-image.jpg" srcset="https://api.gbif.org/v1/image/-KAdqDPkjbN-B6V5aJnpwSQf-SM=/fit-in/2000x0/https%3A%2F%2Fbeaty-biodiversity-museum.hp.gbif-staging.org%2F%2Fasset... 
```

- [ ] 8.
```html
<img src="https://beaty-biodiversity-museum.hp.gbif-staging.org//assets/images/herbarium-card-image.jpg" srcset="https://api.gbif.org/v1/image/zw986ETI4wdTwf7PSKN9GJE_b2Q=/fit-in/2000x0/https%3A%2F%2Fbeaty-biodiversity-museum.hp.gbif-staging.org%2F%2... 
```

- [ ] 9.
```html
<img src="/assets/icons/cc.png" width="24"> 
```

- [ ] 10.
```html
<img src="/assets/icons/by.png" width="24">
```

**Comment by MortenHofft** [comment]
For images related to the theme you can add `imageAlt: some description` in e.g. the frontmatter or in the _data folder for features after you define a background
